packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -245,62 +247,74 @@ class _CodeViewState extends State<CodeView>
       child: Column(
         children: [
           buildCodeviewTitle(theme),
-          DefaultTextStyle(
-            style: theme.fixedFontStyle,
-            child: Expanded(
-              child: Scrollbar(
-                controller: textController,
-                child: ValueListenableBuilder<StackFrameAndSourcePosition>(
-                  valueListenable: widget.controller.selectedStackFrame,
-                  builder: (context, frame, _) {
-                    final pausedFrame = frame == null
-                        ? null
-                        : (frame.scriptRef == scriptRef ? frame : null);
-
-                    return Row(
-                      children: [
-                        ValueListenableBuilder<
-                            List<BreakpointAndSourcePosition>>(
-                          valueListenable:
-                              widget.controller.breakpointsWithLocation,
-                          builder: (context, breakpoints, _) {
-                            return Gutter(
-                              gutterWidth: gutterWidth,
-                              scrollController: gutterController,
-                              lineCount: lines.length,
-                              pausedFrame: pausedFrame,
-                              breakpoints: breakpoints
-                                  .where((bp) => bp.scriptRef == scriptRef)
-                                  .toList(),
-                              executableLines: parsedScript.executableLines,
-                              onPressed: _onPressed,
-                            );
-                          },
-                        ),
-                        const SizedBox(width: denseSpacing),
-                        Expanded(
-                          child: LayoutBuilder(
-                            builder: (context, constraints) {
-                              return Lines(
-                                constraints: constraints,
-                                scrollController: textController,
-                                lines: lines,
-                                pausedFrame: pausedFrame,
-                                searchMatchesNotifier:
-                                    widget.controller.searchMatches,
-                                activeSearchMatchNotifier:
-                                    widget.controller.activeSearchMatch,
-                              );
-                            },
-                          ),
-                        ),
-                      ],
-                    );
-                  },
+          (lines.isNotEmpty)
```

Comment:
instead of using a ternary you can just use inline if / else
```
if (condition)
  widget
else
  widget,
```

Reply:
I tried that before, but the trailing `,` after `widget` was giving me an error (I blame you for conditioning me to put trailing commas after everything!). Seems to work now :-)